examples/dtls-echo/main.c

```diff
@@ -32,7 +32,7 @@
 #error TinyDTLS is configured for working with Sockets. Yet, this is non-socket
 #endif
 
-#define MAIN_QUEUE_SIZE     (8)
+#define MAIN_QUEUE_SIZE     (16)
```

Comment:
This mostly because of the behavior of DTLS while the handshake process is taking place. In some part of it, one peer will transmit five DTLS records in a very short time. Now, because of 6LoWPAN fragmentation, two of them could be fragmented. So, I fear that for the moment the sensors reach the sock_udp_rcv () we could have a full buffer.